p4-16/psa/PSA.mdk

```diff
@@ -512,22 +511,23 @@ translation in the agent software, between target-specific values, and
 a value with a common unit and 0 value reference across all targts.
 ~
 
-~ TBD
-It is not clear yet whether it is reasonable to use a value of type
-`PortId_t` as an index to of a `Register` extern, or of an indexed
-`Counter` or `Meter` extern.  A likely result of attempting to do this
-in a P4 program is that the indexes accessed will be the data plane
-specific values of `PortId_t`, and when the control plane attempts to
-access values in these externs, the numerical translation may not be
-performed on the index.  It is reasonable to use a `DirectCounter` or
-`DirectMeter` with a table key that includes a `PortId_t` field,
-because the agent software will perform port translation for table
-keys.
-
-If this is not a reasonable thing to do, the example program
-`psa-example-counters.p4` should be modified so that it does not do
-this, and scrub all other examples for any similar behavior.
-~
+Note that all of these types have a `p4runtime_translation` annotation
+in the `psa.p4` include file.  This is to ensure that when the
+compiler generates a P4Runtime P4Info file from source programs, it
+will not include the target-specific bit width in the P4Info file.
```

Comment:
it will include the P4Info file the type specified by the `p4runtime_translation` rather than the target-specific bit width.

Reply:
Changed.